src/django/oar/urls.py

```diff
@@ -46,6 +46,10 @@
         name='api_token_auth'),
     url(r'^token-auth-example/', views.token_auth_example,
         name='token_auth_example'),
+    url(r'^contributors/', views.all_contributors, name='all_contributors'),
```

Comment:
What do you think about moving these endpoints under `/api`? Programatic consumers of the data will likely want access to these lists of valid filter options.

Reply:
That sounds good to me -- I'll update the app to make this change.

Reply:
Done in 
90053b6